### ISSUE TITLE
Python 3.10 and Linux arm64 support

### ISSUE THREAD

**Issue by Tenzer** [Azure-Samples/cognitive-services-speech-sdk#1341]
**Describe the bug**
The azure-cognitiveservices-speech package on PyPI is currently neither available for installation on Python 3.10 or the Linux arm64 architecture.

**To Reproduce**
1. Create a virtual environment with Python 3.10 on any of the currently supported architectures or on a Linux arm64 architecture.
2. `pip install azure-cognitiveservices-speech`

**Expected behavior**
A wheel for the current architecture and Python version should be installed.

**Version of the Cognitive Services Speech SDK**
1.19.0

**Platform, Operating System, and Programming Language**
 - OS: [e.g. Windows, Linux, Android, iOS, ...] - We both use macOS and Linux systems
 - Hardware - Both x64 and arm64
 - Programming language: Python 3.9, but would like to switch to 3.10
 - Browser [e.g. Chrome, Safari] (if applicable) - N/A

**Additional context**
`pip` gives the following error message when trying to install the azure-cognitiveservices-speech library:

```
ERROR: Could not find a version that satisfies the requirement azure-cognitiveservices-speech (from versions: none)
ERROR: No matching distribution found for azure-cognitiveservices-speech
```

**Comment by jhakulin** [comment]
@Tenzer Thanks for the report and sorry for the inconvenience as Speech SDK do not yet support for Python version 3.10. We will add the support for that in the next release (1.20.0). Note that currently Speech SDK do not support Python on Linux ARM architecture, that will be also supported in the future, however not yet in 1.20.0. Before the next release is out, could you please use earlier version 3.9 (min version Speech SDK supports at the moment is 3.6) ?

**Comment by Tenzer** [comment]
Yes, we are currently using Python 3.9 but the lack of Python 3.10 support in this library is currently the only thing holding us from upgrading.

**Comment by pankopon** [comment]
Python 3.10 will be supported in the Speech SDK 1.20 release which is expected to be available late January / early February this year. No ETA for Linux arm64 support.

**Comment by BrianMouncer** [comment]
@Tenzer 

I added 3.10 wheels in our [1.20 release](https://pypi.org/project/azure-cognitiveservices-speech/1.20.0/#files).

**Comment by pankopon** [comment]
The Speech SDK 1.24.0 release, which is expected before the end of this year, will add Python packages for Linux ARM64 (on [supported Linux distributions](https://learn.microsoft.com/azure/cognitive-services/speech-service/quickstarts/setup-platform?tabs=linux&pivots=programming-language-python)).

Internal work item ref. 2105397.
